### ISSUE TITLE
stop using system fonts

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1422]
as mentioned in #1420 

we should not use system fonts b/c they look nicer on macs than other platforms, and if we use system fonts, those of us on macs will not see what a big chunk of users see, and will therefore have a mistaken impression of how good things look and how the various fonts and spacings fit togetehr. We should design to one consistent standard, and use a non-system webfont for cases where zilla slab is not appropriate

**Comment by hamilton** [comment]
What is your evidence that this is the case? My understanding is a system font stack (specifically the github font stack, which is the one we're using) is meant to standardize and improve readability on _all_ systems, not just mac. There are sensible defaults used across many OSs as system fonts. From a best practices point of view, this is a pretty common thing recommended by web developers to make sure web properties are actually better-optimized for text.

Always happy to readdress but this feels like it is coming out of nowhere with no real evidence or argument, especially after a year+ of using a native font stack.

**Comment by bcolloran** [comment]
ya, this is something that i've noticed for quite a while, in particular in the context of the report view, which i recall looking noticeably different on my system than on macs, with the non-mac version looks a bit worse. it's not going to make or break us, but i would think it is preferable to make sure we're optimizing a consistent experience for all users.

perhaps this might only really matter for the report view, and system fonts are ok inside of menus and stuff. but i would note that e.g. Medium, which i think we all look to as a content-focused site to follow, uses their own font first, with system fonts as secondary options, and i think we'd want to follow that example minimally for the report view, to make sure we're creating a consistent, great looking report experience for all users.

**Comment by wlach** [comment]
Yeah, I think we should do whatever Medium does for the report view, and something github-ish for the rest of the user interface. I have noted with some annoyance that we're not using the same fonts for the ui in the report view vs. the various server pages.

**Comment by hamilton** [comment]
Yep, this all sounds good to me. What do you mean by "ui in the report view"? Do you mean the full notebook client itself is not using the same font stack, or just the report itself? If so, we should obviously change that (or the server pages). The server pages are still somewhat half-baked (which will need to change down the line but we've been aiming for "yep good enough" on those since the start), but setting the global font for the server pages + notebook is pretty easy to do, and would be worth verifying.

I think there's a deeper question about whether Medium is _the_ paradigm we want to follow. If Iodide were meant to generate prose, I'd have no disagreement. I've been thinking that the next gen of the report view would be closer to Distill than to Medium, but I suspect the two aren't that far away from each other in some ways (though, to be fair, Distill uses a native system font stack for their published papers :) ). Worth considering down the line as part of the effort around #1433.

**Comment by wlach** [comment]
> What do you mean by "ui in the report view"? Do you mean the full notebook client itself is not using the same font stack, or just the report itself?

The notebook client itself. Note these differences between the server pages:

![image](https://user-images.githubusercontent.com/20569/51790282-f5725a80-2160-11e9-9458-81055c45abd8.png)

And the notebook itself:

![image](https://user-images.githubusercontent.com/20569/51790296-2ce10700-2161-11e9-82cb-5b696b3f8169.png)

The difference is subtle but noticeable. I think it's the one in the server pages that we actually want (it seems to be closest to github).

You probably already know this, but github's style system + widgets are actually available for anyone to use: https://styleguide.github.com/primer/ I don't necessarily think we should use it verbatim, but it seems like a good source to borrow ideas from.

**Comment by hamilton** [comment]
@wlach got it, thanks for clarifying / totally agreed. I don't think we're probably setting the system font stack in the client globally so that would account for the difference, since we _are_ on the server pages. The client CSS is still generally all-over-the-place, though slightly less since we landed #1390.

**Comment by bcolloran** [comment]
updated title to reflect discussion -- this really only matters for the report styles, where we want to ensure that content authors and readers will see the same thing across platforms. the editor UI (menus and stuff)using system fonts is fine.
